From 965950416dd09d5441bead5e7602839eb32950b1 Mon Sep 17 00:00:00 2001
From: Jiang Lu <lu.jiang@windriver.com>
Date: Thu, 13 Nov 2014 06:14:55 +0000
Subject: [PATCH 1/8] dpa_offload:Add dependency to prevent linking error

dpa-offload driver invokes routines provided by CAAM driver.
Add dependency for dpa-offload driver to prevent following error
when disable CONFIG_CRYPTO_DEV_FSL_CAAM:

| ERROR: ".caam_jr_enqueue" [drivers/staging/fsl_dpa_offload/dpa_offload.ko] undefined!
| ERROR: ".caam_jr_strstatus" [drivers/staging/fsl_dpa_offload/dpa_offload.ko] undefined!

Signed-off-by: Jiang Lu <lu.jiang@windriver.com>
---
 drivers/staging/fsl_dpa_offload/Kconfig |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/drivers/staging/fsl_dpa_offload/Kconfig b/drivers/staging/fsl_dpa_offload/Kconfig
index baeaab3..4139502 100644
--- a/drivers/staging/fsl_dpa_offload/Kconfig
+++ b/drivers/staging/fsl_dpa_offload/Kconfig
@@ -1,7 +1,7 @@
 
 config FSL_DPA_OFFLOAD
 	tristate "Freescale Datapath Offloading Driver"
-	depends on FSL_FMAN
+	depends on FSL_FMAN && CRYPTO_DEV_FSL_CAAM
 	select CRC8
 	select CRC64_ECMA
 	default n
-- 
1.7.5.4

